HVSample/src/main/res/layout/weight.xml

```diff
@@ -1,41 +1,41 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:orientation="vertical" >
+	android:layout_width="match_parent"
+	android:layout_height="match_parent"
+	android:orientation="vertical" >
 
-    <LinearLayout
-        android:id="@+id/addWeightContainer"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content" >
+	<LinearLayout
+		android:id="@+id/addWeightContainer"
+		android:layout_width="match_parent"
+		android:layout_height="wrap_content" >
 
-        <EditText
-            android:id="@+id/weightInput"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_weight=".8"
-            android:inputType="number"
-            android:text=""
-            android:gravity="left" >
-        </EditText>
+		<EditText
+			android:id="@+id/weightInput"
+			android:layout_width="wrap_content"
+			android:layout_height="wrap_content"
+			android:layout_weight=".8"
+			android:inputType="number"
+			android:text=""
+			android:gravity="left" >
+		</EditText>
 
-        <Button
-            android:id="@+id/addWeight"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_gravity="right"
-            android:layout_weight=".2"
-            android:text="Add" 
-            android:gravity="right"
-            android:textAlignment="center">
-        </Button>
-    </LinearLayout>
+		<Button
+			android:id="@+id/addWeight"
```

Comment:
in other files we were using _ in ids, and here we are camelCase. I am not sure which is right, but which ever is the standard for our SDK, apply to all ids